### PR TITLE
GCW-2335 basic setup for chat

### PR DESCRIPTION
**Pull request by abhinav098**
Hello team, these are the browse side changes for chat between browse and stock for orders.

### CODE REVIEW

File: app/models/order.js [hunk 1]
```diff
@@ -20,7 +21,12 @@ export default Model.extend({
   detailType:       attr('string'),
   districtId:       attr('number'),
   ordersPurposes:     hasMany('ordersPurpose', { async: false }),
+<<<<<<< HEAD
   beneficiaryId: attr('string'),
+=======
```
Comment (steveyken) [inline app/models/order.js]: need to resolve this

Comment (abhinav098) [reply app/models/order.js]: Yupp, my bad, missed this. will do it right away. Thanks @steveyken

File: app/controllers/my_orders.js [hunk 2]
```diff
@@ -17,6 +18,37 @@ export default applicationController.extend({
...
+    return this.get("noMessage") && !this.get("disabled");
+  }),
+
+  sortedMessages: Ember.computed.sort("selectedOrder.messages", "messageSortOptions"),
+
+  groupedMessages: Ember.computed("sortedMessages", function () {
+    return this.groupBy(this.get("sortedMessages"), "createdDate");
+  }),
+
+  groupBy: function (content, key) {
```
Comment (patrixr) [inline app/controllers/my_orders.js]: note: we have lodash installed as a dependency, feel free to use it (e.g `_.groupBy(...)`)   😄

Comment (Darshan-Chauhan) [reply app/controllers/my_orders.js]: Good suggestion

Comment (abhinav098) [reply app/controllers/my_orders.js]: sure @patrixr , i'll try to implement that way.

**Pull request closed without merging**